### PR TITLE
Initial commit...

### PR DESCRIPTION
**Pull request by miltonchandradas**
Simple change of homepage to reflect Jersey Rental and create a video on the process...

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-import logo from './logo.svg';
+
 import './App.css';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+        <img src={require("./jersey.jpg")} width="50%" alt="logo" />
```
Comment (rupakg) [inline src/App.js]: Use the `import` style like the original logo. Apply the styles for the logo to the `App-logo` css class.

**Review by rupakg — COMMENTED** [review]
@miltonc See my comments.

File: src/App.js (hunk 1 above)
Comment (miltonchandradas) [reply src/App.js]: Fixed it...

**Review by rupakg — APPROVED** [review]
Looks good.

**Pull request merged**